### PR TITLE
Refactor test shell

### PR DESCRIPTION
**Pull request by kaikai1024**
### Description of the Change

* Add create_config
* little refactor
* use shellcheck
~~* remove useless jsonrpc test~~

todo: refactor `jsonrpcTest.sh`(the CORS part)

### Applicable Issues

#619

### CODE REVIEW

File: tests/integrate_test/util.sh [hunk 1]
```diff
@@ -323,9 +321,23 @@ start_nodes() {
...
+    ./scripts/create_cita_config.py "$@" > /dev/null 2>&1
+}
+
+create_config() {
+    local param="create \
+        --chain_name $CHAIN_NAME \
+        --super_admin 0x4b5ae4567ad5d9fb92bc9afd6a657e6fa13a2523 \
+        --nodes 127.0.0.1:4000,127.0.0.1:4001,127.0.0.1:4002,127.0.0.1:4003 \
+        $*"
+    # shellcheck disable=SC2086
```
Comment (ouwenkg) [inline tests/integrate_test/util.sh]: 这些注释是啥呀

File: tests/integrate_test/cita_charge_mode.sh [hunk 2]
```diff
@@ -1,113 +1,100 @@
...
+    local version=$1
+    local node_0_privkey
+    local node_1_privkey
+    local node_2_privkey
+    local node_3_privkey
+    node_0_privkey=$(cat ./"$CHAIN_NAME"/0/privkey)
+    node_1_privkey=$(cat ./"$CHAIN_NAME"/1/privkey)
+    node_2_privkey=$(cat ./"$CHAIN_NAME"/2/privkey)
+    node_3_privkey=$(cat ./"$CHAIN_NAME"/3/privkey)
+    cd ./scripts/txtool/txtool
```
Comment (ouwenkg) [inline tests/integrate_test/cita_charge_mode.sh]: local node_0_privkey = $(cat ./"$CHAIN_NAME"/0/privkey) ?

File: tests/integrate_test/cita_bft_resend.sh [hunk 3]
```diff
@@ -2,70 +2,69 @@
...
+    cd "${BINARY_DIR}"
+    echo "DONE"
+
+    echo "1) Generate CITA configurations ..."
+    scripts/create_cita_config.py create \
+        --nodes "127.0.0.1:4000" \
+        --super_admin "${SUPER_ADMIN}" \
+        --chain_name "${CHAIN_NAME}" \
+        --timestamp 1524000000
+    echo "DONE"
```
Comment (ouwenkg) [inline tests/integrate_test/cita_bft_resend.sh]: 这里也可以复用 utils 里的封装吧

Comment (kaikai1024) [reply tests/integrate_test/cita_bft_resend.sh]: This test only need one node running.

File: tests/integrate_test/cita_charge_mode.sh (hunk 2 above)
Comment (kaikai1024) [reply tests/integrate_test/cita_charge_mode.sh]: this will cause warning of shellcheck.

File: tests/integrate_test/util.sh (hunk 1 above)
Comment (kaikai1024) [reply tests/integrate_test/util.sh]: shellcheck wil suggest you use `config_script "$param"`, this's not right. so i ignore it.

**Pull request merged**